terraform/aks/application.tf

```diff
@@ -0,0 +1,39 @@
+module "api_application" {
+
+  source = "./vendor/modules/aks//aks/application"
+
+  is_web = true
+
+  namespace    = var.namespace
+  environment  = var.environment
+  service_name = var.service_name
+
+  cluster_configuration_map = module.cluster_data.configuration_map
+
+  kubernetes_config_map_name = module.application_configuration.kubernetes_config_map_name
+  kubernetes_secret_name     = module.application_configuration.kubernetes_secret_name
+
+
+  docker_image = var.paas_app_docker_image
+  # command     = each.value.startup_command
+  max_memory  = var.memory_max
+  replicas    = var.replicas
+  web_external_hostnames = var.gov_uk_host_names
+  web_port = 8080
+  probe_path = null
+}
+
+module "application_configuration" {
+
+  source = "./vendor/modules/aks//aks/application_configuration"
+
+  namespace             = var.namespace
+  environment           = var.environment
+  azure_resource_prefix = var.azure_resource_prefix
+  service_short         = var.service_short
+  config_short          = var.config_short
+  config_variables      =  { PG_CONN_STR = module.postgres.dotnet_connection_string, AKS_ENV_NAME = var.file_environment, REDIS_CONN_STR = module.redis-cache.connection_string, EnableMetrics = false}
```

Comment:
These connection strings should be included in the secrets as they contain the credentials needed to connect to the databases.